tests/user-test.js

```diff
@@ -68,7 +69,7 @@ describe('User', () => {
   });
 
   it.skip('Should be able to check ingredients in User/s pantry for a given recipe', () => {
-    user1.;
```

Comment:
I'm not sure why I let `user1.;` through on the PR, apologies, I was losing focus late last night.